python/delete_unlinked_top_containers.py

```diff
@@ -0,0 +1,62 @@
+import asnake.logging as logging
+from asnake.client import ASnakeClient
+import argparse
+
+logging.setup_logging(filename="archivessnake.log", level="INFO")
+# set label for custom logger - all output will be in archivessnake.log
+logger = logging.get_logger("delete_unlinked_top_containers")
+client = ASnakeClient()
+
+
+def container_is_unlinked(top_container_uri: str) -> bool:
+    top_container = client.get(top_container_uri).json()
+    if len(top_container["collection"]) == 0:
```

Comment:
A more Pythonic way to do this:
```return len(top_container["collection"]) == 0```

Since there's no logging or complex logic here, I recommend the one-liner replacing the conditional.